pkg/pillar/cmd/usbmanager/usbcontroller.go

```diff
@@ -224,7 +224,23 @@ func (uc *usbmanagerController) cancel() {
 func ioBundle2PassthroughRule(adapter types.IoBundle) passthroughRule {
 	var pr passthroughRule
 
-	if adapter.UsbAddr == "" && adapter.PciLong != "" {
+	countAddr := 0
+
+	if adapter.UsbAddr != "" {
+		countAddr++
+	}
+
+	if adapter.UsbProduct != "" {
+		countAddr++
+	}
+
+	if countAddr > 1 {
+		log.Warnf("ambiguous passthrough rule %s (usbaddr: %s) (usbproduct: %s) (pcilong: %s)",
```

Comment:
Is this the semantics we want, as opposed to matching all that is specified?
If the pcilong is specified, do you match on that plus the usbaddr/usbproduct?

Reply:
> Is this the semantics we want, as opposed to matching all that is specified?

I will try it out; it does not look hard to implement it that way.

> Is this the semantics we want, as opposed to matching all that is specified?

Currently yes, but it will change once I use the `parentassigngrp`